nixos/modules/system/boot/loader/generic-extlinux-compatible/extlinux-conf-builder.sh

```diff
@@ -1,4 +1,4 @@
-#! @bash@/bin/sh -e
+#! @bash@/bin/bash -e
```

Comment:
This uses more bash-isms now so I changed the shebang to be clear. (Stuff like nullglob was already a bash-ish though)

Reply:
`@bash@` is obvious already but I think there can be differences in bash's behaviour if invoked as `sh` instead of `bash`, so yeah, that won't hurt.